lib/components/screens/search/search_screen.dart

```diff
@@ -1,63 +1,24 @@
-import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:harpy/components/components.dart';
-import 'package:harpy/core/core.dart';
-import 'package:harpy/harpy_widgets/harpy_widgets.dart';
-import 'package:harpy/misc/misc.dart';
-import 'package:provider/provider.dart';
 
 class SearchScreen extends StatelessWidget {
-  const SearchScreen();
+  const SearchScreen({
+    Key? key,
+  }) : super(key: key);
```

Comment:
```suggestion
  const SearchScreen();
```
Let's not add a key param if we don't need it

Reply:
Done